plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesExecutor.groovy

```diff
@@ -74,23 +83,74 @@ class TesExecutor extends Executor implements ExtensionPoint {
         remoteBinDir
     }
 
+    @PackageScope
+    List<Path> getRemoteBinFiles() {
+        remoteBinFiles
+    }
+
     protected void uploadBinDir() {
         /*
          * upload local binaries
          */
         if( session.binDir && !session.binDir.empty() && !session.disableRemoteBinDir ) {
-            def tempBin = getTempDir()
+            final tempBin = getTempDir()
             log.info "Uploading local `bin` scripts folder to ${tempBin.toUriString()}/bin"
             remoteBinDir = FilesEx.copyTo(session.binDir, tempBin)
+
+            remoteBinFiles = []
+            session.binDir.eachFileRecurse { file ->
+                if( file.isDirectory() )
+                    return
+                remoteBinFiles << tempBin.resolve(session.binDir.relativize(file).toString())
```

Comment:
@bentsherman 

This is not getting the correct url of the file path to the tes task.

`nftes3dd92310def6b/work/tmp/c5/993adc8e95593d80e56469aef20d78/run.sh`
but should be
`nftes3dd92310def6b/work/tmp/c5/993adc8e95593d80e56469aef20d78/bin/run.sh`